.github/workflows/js-checks.yml

```diff
@@ -0,0 +1,15 @@
+name: JS Checks
+
+on:
+  pull_request:
+  push:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  js-checks:
+    uses: hemilabs/actions/.github/workflows/js-checks.yml@main
+    with:
+      node-versions: '["16", "18", "20", "22"]'
```

Comment:
Just checking but do we really care that far back?  node16?

Reply:
I was just following the standard set by @gabmontes  in other packages (See for example [this one](https://github.com/hemilabs/crypto-shortener/blob/1605736ff15077ac1651b192fabc01645498795b/.github/workflows/js-checks.yml#L15)). 

I'm open to ignoring node16; node18 seems a good compromise.

Either way, this package is very simple so it should work for even older versions

Reply:
I was thinking more that anything that discourages the use of node16 is a good thing.  But consistency is definitely best.